src/components/shared/SearchCard/SearchCardFooter.js

```diff
@@ -2,22 +2,22 @@ import { useRouter } from 'next/router'
 import { Box } from 'grommet'
 import { useSearchManager } from 'hooks/useSearchManager'
 import formatURLString from 'helpers/formatURLString'
+import gtag from 'analytics/gtag'
 import { Button } from 'components/shared/Button'
 
 export const SearchCardFooter = ({ accessionCode, title }) => {
   const { push } = useRouter()
   const { search, setSearch } = useSearchManager()
 
+  const handleClick = () => {
+    push(`/experiments/${accessionCode}/${formatURLString(title)}`)
+    setSearch({ ...search, ref: 'search' })
+    gtag.trackExperimentPageClick(SearchCardFooter)
```

Comment:
This tracking method was missing from this component, so I've included.

Reply:
Can we move the `push` to the end